examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h

```diff
@@ -45,6 +45,8 @@ class CHIPCommandBridge : public Command {
         AddArgument("paa-trust-store-path", &mPaaTrustStorePath,
             "Path to directory holding PAA certificate information.  Can be absolute or relative to the current working "
             "directory.");
+        AddArgument("storage-directory", &mStorageDirectory,
+            "Directory to place chip-tool's storage files in.  Defaults to $TMPDIR, with fallback to /tmp");
```

Comment:
```suggestion
            "Directory to place darwin-framework-tool's storage files in.  Defaults to $TMPDIR, with fallback to /tmp");
```
But is this really true?  If we are only doing this for interactive commands, I fully expect this thing is pretty much ignored.....

Reply:
You are correct that this is not really true at the moment. The current storage mechanism use some built-in preference system. I wonder if I should:
 - add a custom persistent storage backend that stores things in files. But I think that was an issue to run GitHub workflows and that is why this thing as been implemented using preferences. @krypton36  ?
 - update the mechanism to works with a domain for `darwin-framework-tool`. For example `--storage-directory` may change the domain used to save preferences (from `com.apple.chiptool` to `com.apple.chiptool-{storage-directory}`)

Reply:
In any case this should probably be a followup unless this is causing issues. I would like to move forward and try to get https://github.com/vivien-apple/connectedhomeip-1/tree/MatterYamlTests_DarwinFrameworkTool_All_Tests in.

The mentioned branch gets the command `/scripts/tests/run_test_suite.py --runner chip_tool_python --chip-tool out/debug/standalone/darwin-framework-tool  run --test-timeout-seconds 120` to works successfully and it depends on this one ;)

Reply:
I think for now accurately documenting the option as "it's here for API compat with chip-tool but does nothing" is the thing to do.